main.go

```diff
@@ -56,7 +57,14 @@ func init() {
 		&localPkgPrefixes,
 		localPkgPrefixesArg,
 		"",
-		`Local package prefixes which will be placed after 3rd-party group(if defined). Values should be comma-separated. Optional parameters.`,
+		"Local package prefixes which will be placed after 3rd-party group(if defined). Values should be comma-separated. Optional parameters.",
+	)
+
+	flag.StringVar(
+		&output,
+		outputArg,
+		"file",
+		`Can be "file" or "stdout". Whether to write the formatted content back to the file or to stdout. Optional parameter.`,
```

Comment:
Mark which one is by default (same as for README.md)

Reply:
Hmm so I initially did that, but it actually gets added automatically when printing the usage (the stuff in the README is just from the usage print). I think it's a kingpin feature. Maybe try checking out the branch and running it to see if it's what you want? If I add it manually, it'll show up double.

Reply:
sorry, you’re right